### PR TITLE
Add numeric type in the doc [ci skip]

### PR DESCRIPTION
**Pull request by kamipo**
Follow up to #23508.

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb [hunk 1]
```diff
@@ -477,9 +477,9 @@ def drop_table(table_name, options = {})
       #   Allows or disallows +NULL+ values in the column. This option could
       #   have been named <tt>:null_allowed</tt>.
       # * <tt>:precision</tt> -
-      #   Specifies the precision for a <tt>:decimal</tt> column.
+      #   Specifies the precision for <tt>:decimal</tt> and <tt>:numeric</tt> columns.
```
Comment (maclover7) [inline activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb]: precision for the

File: activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb [hunk 2]
```diff
@@ -477,9 +477,9 @@ def drop_table(table_name, options = {})
...
       # * <tt>:scale</tt> -
-      #   Specifies the scale for a <tt>:decimal</tt> column.
+      #   Specifies the scale for <tt>:decimal</tt> and <tt>:numeric</tt> columns.
```
Comment (maclover7) [inline activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb]: precision for the

**Comment by maclover7** [comment]
Small grammar notes, then ready to merge.

**Comment by kamipo** [comment]
I fixed grammar, thanks!

**Comment by vipulnsward** [comment]
Thanks @kamipo !

**Comment by kamipo** [comment]
:smile:

**Pull request merged**